nvdaHelper/vbufBackends/gecko_ia2/gecko_ia2.cpp

```diff
@@ -815,21 +803,21 @@ VBufStorage_fieldNode_t* GeckoVBufBackend_t::fillVBuf(IAccessible2* pacc,
 	}
 
 	// Add some presentational table attributes
-	// Note these are only for reporting, the physical table attributes (table-physicalrownumber etc) for aiding in navigation etc are added  later on.
+	// Note these are only for reporting, the physical table attributes (table-rownumber etc) for aiding in navigation etc are added  later on.
 	// propagate table-rownumber down to the cell as Gecko only includes it on the row itself
 	if(parentPresentationalRowNumber) 
-		parentNode->addAttribute(L"table-rownumber",parentPresentationalRowNumber);
+		parentNode->addAttribute(L"table-rownumber-presentational",parentPresentationalRowNumber);
 	const wchar_t* presentationalRowNumber=NULL;
 	if((IA2AttribsMapIt = IA2AttribsMap.find(L"rowindex")) != IA2AttribsMap.end()) {

```

Comment:
While here, I would love it if you could change these conditions into two statements:
```suggestion
	IA2AttribsMapIt = IA2AttribsMap.find(L"rowindex");
	if(IA2AttribsMapIt != IA2AttribsMap.end()) {
```